uvloop/__init__.py

```diff
@@ -34,10 +37,16 @@ def install() -> None:
     __asyncio.set_event_loop_policy(EventLoopPolicy())
 
 
-def run(main, *, loop_factory=new_event_loop, debug=None, **run_kwargs):
+def run(
+    main: _typing.Coroutine[_typing.Any, _typing.Any, _T],
+    *,
+    loop_factory: _typing.Optional[_typing.Callable[[], Loop]] = new_event_loop,
+    debug: _typing.Optional[bool]=None,
+    **run_kwargs: object
```

Comment:
this is a bit problematic with mypy typing, it's a lie that run_kwargs exists but it's needed for backwards compatibility

Reply:
might be best to do this whole thing in an:

```
if _typing.TYPE_CHECKING:
    def run(...): ...
else:
    def run(..., **run_kwargs): ...
```